From 7c0ba83a2c87a03473696d5e2a23e11050adde55 Mon Sep 17 00:00:00 2001
From: Zygmunt Krynicki <zygmunt.krynicki@canonical.com>
Date: Thu, 8 Oct 2015 08:38:22 -0700
Subject: Revert the documentation theme back to default

 CheckboxNG uses a customized sphinx theme that is not included in the release
 tarball. The theme is customized for "ubuntu" colors and this is not required
 in Debian.
Origin: upstream
Forwarded: not-needed
Last-Update: 2015-07-21

Patch-Name: documentation-theme
---
 docs/conf.py | 24 ++----------------------
 1 file changed, 2 insertions(+), 22 deletions(-)

diff --git a/docs/conf.py b/docs/conf.py
index 9806b35..29643a6 100644
--- a/docs/conf.py
+++ b/docs/conf.py
@@ -127,20 +127,7 @@ pygments_style = 'sphinx'
 # -- Options for HTML output -------------------------------------------------
 
 # html_theme_path = ['_templates']
-html_theme = 'alabaster'
-html_theme_options = {
-	'github_button': 'false',
-    'page_width': '1040px',
-    'sidebar_width': '320px',
-    'sidebar_link': '#E95420',
-    'sidebar_link_underscore': '#E95420',
-    'link': '#E95420',
-    'link_hover': '#E95420',
-    'font_family': "'Ubuntu', sans-serif",
-    'head_font_family': "'Ubuntu', sans-serif",
-    'code_font_family': "'Ubuntu Mono', monospace",
-    'code_font_size': '1em'
-    }
+html_theme = 'default'
 
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
@@ -183,14 +170,7 @@ html_theme_options = {
 # html_use_smartypants = True
 
 # Custom sidebar templates, maps document names to template names.
-html_sidebars = {
-    '**': [
-        'about.html',
-        'navigation.html',
-        'relations.html',
-        'searchbox.html'
-    ]
-}
+# html_sidebars = {}
 
 # Additional templates that should be rendered to pages, maps page names to
 # template names.
